discourse-mixin/panels.libsonnet

```diff
@@ -0,0 +1,148 @@
+        targets=[signals.overview.activeRequests.asTarget()]
+      )
+      + g.panel.timeSeries.panelOptions.withDescription('Active web requests for the entire application')
+      + g.panel.timeSeries.standardOptions.withUnit('reqps'),
+
+    queuedRequests:
+      commonlib.panels.network.timeSeries.traffic.new(
+        'Queued requests',
+        targets=[signals.overview.queuedRequests.asTarget()]
+      )
+      + g.panel.timeSeries.panelOptions.withDescription('Queued web requests for the entire application.')
+      + g.panel.timeSeries.standardOptions.withUnit('reqps'),
+
+    pageViews:
+      commonlib.panels.network.timeSeries.traffic.new(
+        'Page views',
+        targets=[signals.overview.pageViews.asTarget()]
+      )
+      + g.panel.timeSeries.panelOptions.withDescription('Rate of pageviews for the entire application. Grouped by type and service.')
+      + g.panel.timeSeries.standardOptions.withUnit('none'),
+
+    latestMedianRequestTime:
+      commonlib.panels.network.timeSeries.base.new('Latest median request time', targets=[signals.overview.latestMedianRequestTime.asTarget()])
+      + g.panel.timeSeries.panelOptions.withDescription('The median amount of time for "latest" page requests for the selected site.')
+      + g.panel.timeSeries.standardOptions.withUnit('s'),
+
+    topicMedianRequestTime:
+      commonlib.panels.network.timeSeries.base.new('Topic show median request time', targets=[signals.overview.topicMedianRequestTime.asTarget()])
+      + g.panel.timeSeries.panelOptions.withDescription('The median amount of time for "topics show" requests for the selected site.')
+      + g.panel.timeSeries.standardOptions.withUnit('s'),
+
+    latest99thPercentileRequestTime:
+      g.panel.histogram.new('Latest request time')
+      + g.panel.histogram.queryOptions.withTargets([signals.overview.latest99thPercentileRequestTime.asTarget()])
+      + g.panel.histogram.panelOptions.withDescription('The 99th percentile amount of time for "latest" page requests for the selected site.')
+      + g.panel.histogram.standardOptions.withUnit('s'),
+
+    topic99thPercentileRequestTime:
+      g.panel.histogram.new('Topic show request time')
+      + g.panel.histogram.queryOptions.withTargets([signals.overview.topic99thPercentileRequestTime.asTarget()])
+      + g.panel.histogram.panelOptions.withDescription('The amount of time for "topics show" requests for the selected site.')
+      + g.panel.histogram.standardOptions.withUnit('s'),
+
+    // Jobs dashboard panels
+    sidekiqJobDuration:
+      commonlib.panels.generic.timeSeries.base.new('Sidekiq job duration', targets=[signals.jobs.sidekiqJobDuration.asTarget()])
+      + g.panel.timeSeries.options.legend.withAsTable(true)
+      + g.panel.timeSeries.options.legend.withPlacement('right')
+      + g.panel.timeSeries.panelOptions.withDescription('Time spent in Sidekiq jobs broken out by job name.')
+      + g.panel.timeSeries.standardOptions.withUnit('s'),
```

Comment:
I wonder if this panel would look better with stacking enabled? Since we want to show cumulative job duration and breakdown by job.